aws/sqldatabases.bicep

```diff
@@ -50,6 +50,9 @@ param allocatedStorage string = '20'
 @description('Database license model')
 param licenseModel string = 'license-included'
 
+@description('The number of days for which automated backups are retained. Setting this parameter to a positive number enables backups. Defaults to 0.')
+param backupRetentionPeriod int = 0
```

Comment:
Can we set the default to 1 (same as what RDS sets https://docs.aws.amazon.com/cli/latest/reference/rds/create-db-instance.html), and override it to zero for our eshop scenario?